sydent/threepid/bind.py

```diff
@@ -101,29 +102,27 @@ def _notify(self, assoc, attempt):
         mxid = assoc["mxid"]
         domain = mxid.split(":")[-1]
```

Comment:
this is the wrong way to extract a domain from an mxid, ftr... but that's probably a problem for another day.